src/main/webapp/style.css

```diff
@@ -1,8 +1,22 @@
+#emphasis {
+  color: violet;
+}
+
 h1 {
-  font-family: Verdana, sans-serif; 
-  font-size: x-large;
+  margin-top: 100px; 
 }
 
-#emphasis {
-  color: violet;
+#form_button {
+  background-color: violet;
+  border: none;
+  border-radius: 8px;
+  font-size: xx-large;
+  left: 50%;
+  margin-top: 200px;
+  text-align: center;
+  width: 70%;
 }
```

Comment:
Please look to use Bootstrap where at all possible with styling. Consider using `mt-4` or similar for margins. And `text-center` for centering text. Also consider maybe using the existing [Bootstrap buttons](https://getbootstrap.com/docs/4.0/components/buttons/) and overriding the color differences for consistency. I plan to use Bootstrap buttons so we should aim to keep it consistent.

Reply:
The Bootstrap buttons are very limited. I wanted to stay consistent with our color scheme, and think the button is much more appealing with the curved corners and shadow on hover. That is why I did not choose to use a Bootstrap button.

Reply:
Also, is it not nicer to have a single, well-defined form_button id rather than stacking a bunch of Bootstrap classes (a basic bootstrap button, mt-4, text-center) and my own modifications?

Reply:
hmmm. I think it would be worth it to have a default button theme. I think Richie is trying to avoid losing out on some of the bootstrap functionality. If we use the ```btn``` class and add our own class with the modifications that it doesn't have.
for instance;
```
button.btn {
  border-radius: 8px;
  text-align: center;
}
```
You could add the ```!important``` tag to override bootstrap if the changes don't show too.

Reply:
Thank you so much @Obi! That sounds like an awesome plan.

Reply:
I am going to push a new version. Let me know if this is what you intended @Obinnabii

Reply:
yeah it looks good! does it work when you preview?